notary-server/config.json

```diff
@@ -0,0 +1,19 @@
+{
+	"server": {
+		"addr": ":4443"
+	},
+	"trust_service": {
+	  "type": "remote",
+	  "hostname": "notarysigner",
+	  "port": "7899",
+	  "tls_ca_file": "./fixtures/root-ca.crt",			
+	  "key_algorithm": "ecdsa"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "mysql",
+		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
```

Comment:
if these are going to be hardcoded like this, the recommended way to run this image should be to bind mount as :ro the config.json